src/components/dashboard/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Box, Grid } from '@mui/material';
+import TenantSelector from 'components/shared/TenantSelector';
+import {
+    semiTransparentBackground_blue,
+    semiTransparentBackground_purple,
+    semiTransparentBackground_teal,
+} from 'context/Theme';
+import { CloudDownload, CloudUpload, DatabaseScript } from 'iconoir-react';
+import EntityStatOverview from './EntityStatOverview';
+import useMonthlyUsage from './EntityStatOverview/useMonthlyUsage';
+
+const ICON_SIZE = 12;
+
+export default function Dashboard() {
+    const { captureUsage, loading, materializationUsage } = useMonthlyUsage();
+
+    return (
+        <Grid container spacing={{ xs: 4 }} style={{ marginTop: 32 }}>
+            <Grid
+                item
+                xs={12}
+                sx={{ display: 'flex', justifyContent: 'flex-end' }}
+            >
+                <Box style={{ width: 300 }}>
+                    <TenantSelector />
+                </Box>
+            </Grid>
+
+            <Grid item xs={4}>
+                <EntityStatOverview
+                    Icon={<CloudUpload fontSize={ICON_SIZE} />}
```

Comment:
Might be over engineering... but should we make some kind of shared "config" for entities where we store off stuff like what icon is shown for them? Might be too soon since it is only used in two spots right now.